src/main/main.ts

```diff
@@ -667,27 +663,34 @@ async function initializeStandalone(isRestart: boolean = false): Promise<void> {
       );
     } else if (snapshotPaths.length > 0 && win != null) {
       const snapshotDownloadUrls: string[] = electronStore.get("SnapshotPaths");
-      if (snapshotDownloadUrls.length > 0 && win != null && !isRestart) {
-        for (const snapshotDownloadUrl of snapshotDownloadUrls) {
-          const isProcessSuccess = await snapshot.processSnapshot(
-            snapshotDownloadUrl,
-            BLOCKCHAIN_STORE_PATH,
-            app.getPath("userData"),
-            standalone,
-            win,
-            initializeStandaloneCts.token
-          );
-          if (isProcessSuccess) break;
-        }
+      let isProcessSuccess = false;
+      for (const snapshotDownloadUrl of snapshotDownloadUrls) {
+        isProcessSuccess = await snapshot.processSnapshot(
```

Comment:
it can't be a big deal... but IMHO, handling an exception from `processSnapshot()` can be better than using boolean return value because we can get more detailed information about the exception.

Reply:
Ok, I'll check try-catch on `main` not `snapshot`

Reply:
7585794 changed